000-cbdb-sandbox/Dockerfile.RELEASE.centos7

```diff
@@ -13,15 +18,15 @@ COPY ./configs/* /tmp/
 RUN     echo root:cbdb@123 | chpasswd \
         && yum install -y epel-release \
         && yum install -y initscripts unzip which tar less net-tools util-linux-ng passwd openssh-clients openssh-server perl ed m4 sudo rsync git wget \
-        && yum install -y apr-devel bison bzip2-devel cmake3 flex gcc gcc-c++ krb5-devel libcurl-devel libevent-devel libkadm5 libxml2-devel libyaml-devel libzstd-devel openssl-devel perl-ExtUtils-Embed readline-devel xerces-c-devel zlib-devel \
+        && yum install -y apr-devel bison bzip2-devel cmake3 flex gcc gcc-c++ go krb5-devel libcurl-devel libevent-devel libkadm5 libxml2-devel libyaml-devel libzstd-devel openssl-devel perl-ExtUtils-Embed readline-devel xerces-c-devel zlib-devel \
```

Comment:
Hi @edespino, I think we can skip support for Cloudberry 1.5.3, as this release requires additional dependencies like Golang. However, it has been refactored in the following pull request. We will release 1.5.4 this month. Does it make sense?

Reply:
I think it is good to keep the installation of go. I will be introducing the building of gpbackup & gprestore which will need to. These are basic utilities which are discussed in the Bootcamp.

Reply:
Thanks, that makes sense. 👍